### ISSUE TITLE
How do get car mask only?

### ISSUE THREAD

**Issue by yfnn** [isht7/pytorch-deeplab-resnet#33]
I used your code for car segmentation. I loaded the pretrained model. But there are 21 labels in VOC. I just want to get mask of cars while not display other labels on the result image. What should I do?

**Comment by isht7** [comment]
If the label for car is x, you can get a binary mask with 1 for car and 0 for everything else by doing 
`mask = output_pred==x`
